src/main/java/com/faforever/client/replay/Replay.java

```diff
@@ -179,11 +184,11 @@ public Temporal getStartTime() {
     return startTime.get();
   }
 
-  public void setStartTime(Temporal startTime) {
+  public void setStartTime(OffsetDateTime startTime) {
```

Comment:
No idea why we have this method, but only the one with `Temporal` needs to exist. We can remove this one.